webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -35,29 +37,140 @@ function GithubItems(props) {
                 </a>);
         }
 
+        let milestone = '';
+        if (item.milestone) {
+            milestone = (<span><i className='fas fa-bullseye'/>{item.milestone.title}</span>);
+        }
+
+        let reviewers = '';
+        if (item.reviewers && item.reviewers > 0) {
+            reviewers = (<span>{item.reviewers} pending reviews.</span>);
```

Comment:
I believe these are the amount of reviewers requested regardless if pending or not. Not sure if there is any value on providing the total reviewers. 

Thoughts @mickmister ?

Reply:
Could we have a `3 out of 4 reviews complete` display or something of the sort?

Reply:
This is a mistake on my part. The reviewers value comes from "requested_reviewers". As far as I have seen, a requested reviewer is marked as that since the it is requested for review until it makes its first review. After that, even if you request a rereview, it will not appear on "requested_reviewers" anymore.

That being said... reviewers is indeed the pending reviews. Also, because of that, my first question about how to deal with the reviews. I can "do the maths" and leave it as `3 out of 4 reviews complete`, or I could go even overkill with something as:
```
- 1 approved review.
- 2 changes requested reviews.
- 1 pending review.
- 1 out of 4 reviews complete.
```

What I cannot do (as far as I have seen at least) is distinguish between a Changes Requested review and a rerequested review.

All that being said, what do you expect to be the most useful way to give this information?

Reply:
@larkox I think saying `1 out of 4 reviews complete` and a small indicator to express that at least one reviewer currently has requested changes (just like the red icon that shows up in GitHub) would be sufficient. I think this solution is both compact and informative.